src/server/test-suites/helpers/provide-tabs.test.js

```diff
@@ -0,0 +1,285 @@
+import { provideTabs } from '~/src/server/test-suites/helpers/provide-tabs.js'
+
+const mockRouteLookup = jest.fn()
+const mockUserIsServiceOwner = jest.fn()
+const mockUserSession = jest.fn()
+const mockRequest = ({ response, path = '' }) => ({
+  response,
+  path,
+  getUserSession: mockUserSession,
+  userIsServiceOwner: mockUserIsServiceOwner,
+  routeLookup: mockRouteLookup
+})
+
+describe('#provideTabs', () => {
+  const mockServiceName = 'cdp-env-test-suite'
+  const mockSource = {
+    context: {
+      service: {
+        imageName: `${mockServiceName}`
+      }
+    }
+  }
+  const mockResponse = {
+    variety: 'view',
+    source: mockSource
+  }
+  const mockViewHelper = {
+    continue: 'mockContinue'
+  }
+
+  afterEach(() => {
+    mockResponse.source = mockSource
+  })
+
+  describe('With an Admin user', () => {
+    beforeEach(() => {
+      mockUserIsServiceOwner.mockResolvedValue(false)
+      mockRouteLookup.mockImplementation(
+        (path, options) =>
+          '/' + path.replace('{serviceId}', options.params.serviceId)
+      )
+      mockUserSession.mockResolvedValue({
+        isAdmin: true,
+        isTenant: true
+      })
+    })
+
+    test('Should provide expected context tabs', async () => {
+      await provideTabs(
+        mockRequest({
+          response: mockResponse,
+          path: `/test-suites/${mockServiceName}`
+        }),
+        mockViewHelper
+      )
+
+      expect(mockResponse.source.context.tabDetails.tabs).toEqual([
+        {
+          isActive: true,
+          label: 'About',
+          url: `/test-suites/${mockServiceName}`
+        },
+        // {
```

Comment:
please remove

Reply:
Sure, but gonna uncomment it all in next PR :-(